### PR TITLE
add lay

### PR DESCRIPTION
**Pull request by relfor**

### CODE REVIEW

**Comment by relfor** [comment]
#1553

File: examples/lay/bower.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "LAY": "latest"
+  }
+}
```
Comment (sindresorhus) [inline examples/lay/bower.json]: Use package.json and npm instead.

File: examples/lay/index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (sindresorhus) [inline examples/lay/index.html]:
```
<!doctype html>
```

File: examples/lay/index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<html>
+<head>
+<meta charset="utf-8"/>
```
Comment (sindresorhus) [inline examples/lay/index.html]:
```html
<meta charset="utf-8">
```

File: examples/lay/index.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
```
Comment (sindresorhus) [inline examples/lay/index.html]:
```html
<meta name="viewport" content="width=device-width, initial-scale=1">
```

File: examples/lay/index.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+</head>
+<body>
+</body>
+<script type="text/javascript" src="bower_components/LAY/LAY.min.js"></script>
```
Comment (sindresorhus) [inline examples/lay/index.html]: `type` is not needed

File: examples/lay/index.html [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+<script type="text/javascript" src="index.js"></script>
+</html>
```
Comment (sindresorhus) [inline examples/lay/index.html]: Properly indent the HTML tags.

**Comment by addyosmani** [comment]
Thanks for sending over this PR. My two cents: implementations that use non-mainstream stacks provide value in a "this is an interesting approach to the problem space", but are better referenced from our wiki than the main site. We'll get this added there.

**Pull request closed without merging**